webapp/src/js/controllers/reports.js

```diff
@@ -336,6 +336,7 @@ angular
         }
 
         doc.verified = doc.verified === valid ? undefined : valid;
+        doc.verified_date = Date.now();
```

Comment:
Contrary to what the PR title says, this will set `verified_date` _every time_ the doc is verified.

Reply:
It'll also update the date when the report is unverified which would be confusing. Maybe we should wipe the date when it's unverified?

Reply:
I'll follow up with Muso data analytics team. In our last meeting they said they would like to track the "last" signoff time.